make/photon/db/docker-entrypoint.sh

```diff
@@ -1,6 +1,18 @@
 #!/bin/bash
 set -e
 
+PG_VERSION_NEW=$1
```

Comment:
`PG_VERSION_NEW` => `PG_VERSION` ? From the point of the `harbor-db` , 13 is the current version of the `harbor-db` not the`new` version.